### ISSUE TITLE
Applying Species Bee Genetic Crashes Game

### ISSUE THREAD

**Issue by Evansj306** [JDKDigital/productive-bees#449]
When attempting to apply a productive bee with any species genetic trait on a honey treat, the game crashes. In my instance and the original issue was with vanilla bee genetics.

![2023-12-28_12 14 04](https://github.com/JDKDigital/productive-bees/assets/42954991/8dd0f8af-2f4f-4abc-8f7b-3ae5d9f7ecff)

**Comment by Evansj306** [comment]
Apologies, here is the log file in full:
[crash-2023-12-28_12.20.20-server.txt](https://github.com/JDKDigital/productive-bees/files/13788299/crash-2023-12-28_12.20.20-server.txt)

**Comment by Evansj306** [comment]
Looking further into this, it looks like your not including the species attribute in your attribute map under util/BeeAttributes.java.

**Comment by JaisDK** [comment]
It's not supposed to be in that list and you're not supposed to be able to combine a type gene with the other ones. What crafting grid did you use to make it?
You can't change a bees type with a treat if this is what you're trying to do. You use the type genes on their own treat in an incubator with an egg to make spawn eggs.

**Comment by Evansj306** [comment]
That was exactly what I was trying to do. :) 

I crafted it using the refined storage (RebornStorage) Super Wireless Grid on the Direworlf 20 1.20 FTB pack. 

Interesting, I tried a productive bee type and that beehaves the way you described. It might be only the vanilla bee type that can be combined with the other attributes then.
![2023-12-29_14 58 13](https://github.com/JDKDigital/productive-bees/assets/42954991/8e0a87d1-94f0-486a-85d1-ac3d04ff00c0)

If that's the case, should this recibee be disabled as well? Ie applying a type to a caged bee?
![2023-12-29_15 01 08](https://github.com/JDKDigital/productive-bees/assets/42954991/b98f5ee9-947e-4f1a-803d-fe32b0b2907e)

ps, love the bee puns. 

Thanks!
